stdlib/public/Darwin/Foundation/String.swift

```diff
@@ -49,15 +102,82 @@ extension String : _ObjectiveCBridgeable {
     self._forceBridgeFromObjectiveC(x, result: &result)
     return result != nil
   }
+  
+  @_effects(releasenone)
+  @inline(__always)
+  private static func _bridgeToSmall(
+    _ source: NSString,
+    _ len: Int,
+    _ encoding: CFStringBuiltInEncodings) -> String? {
```

Comment:
I don't think this should be parameterized over encoding. This is incorrect for anything that's not UTF8/ASCII.

Reply:
It's just so I could pass UTF8 in one place and ASCII in the other. I can make that restriction more apparent though.